frontend/src/components/Join.tsx

```diff
@@ -41,6 +45,10 @@ class Join extends React.Component<Props, State> {
     this.setState({ id: e.target.value });
   }
 
+  handleUserNameCreation(e: any) {
```

Comment:
Should probably change this name as well

Reply:
Refactored! :D